### PR TITLE
Making sure hanging request functionality is actually being used

### PR DESCRIPTION
**Pull request by mlunoe**
Before:

After:

### CODE REVIEW

File: utils/RequestUtil.js [hunk 1]
```diff
@@ -71,13 +71,10 @@ var RequestUtil = {
   },
 
   json: function (options) {
+    var usingHangingRequest = Util.isFunction(options.hangingRequestCallback);
     if (options) {
-      if (options.url && !/\?/.test(options.url)) {
-        options.url += "?_timestamp=" + Date.now();
-      }
```
Comment (rcorral) [inline utils/RequestUtil.js]: lol well that was a huge mistake

**Comment by rcorral** [comment]
Awesome work!

File: utils/RequestUtil.js [hunk 2]
```diff
@@ -71,13 +71,10 @@ var RequestUtil = {
   },
 
   json: function (options) {
+    var usingHangingRequest = Util.isFunction(options.hangingRequestCallback);
     if (options) {
```
Comment (jfurrow) [inline utils/RequestUtil.js]: Do we need this? We are assuming `options` is defined in the line above this check.

File: utils/RequestUtil.js (hunk 1 above)
Comment (jfurrow) [inline utils/RequestUtil.js]: What is the mistake here?

File: utils/RequestUtil.js (hunk 2 above)
Comment (mlunoe) [inline utils/RequestUtil.js]: Fixed by default assigning `options` to an empty object. This means lines were unindented

**Pull request merged**